test/security/ClientSSLSecurity.js

```diff
@@ -49,4 +49,39 @@ describe('ClientSSLSecurity', function() {
     var instance = new ClientSSLSecurity(null, null, caList);
     instance.should.have.property("ca", caList);
   });
+
+  describe('forever parameter', function () {
+    it('should generate different agents if paramter not given', function () {
+      var instance = new ClientSSLSecurity();
+      var firstOptions = {};
+      var secondOptions = {};
+
+      instance.addOptions(firstOptions);
+      instance.addOptions(secondOptions);
+
+      firstOptions.agent.should.not.equal(secondOptions.agent);
+    });
+
+    it('should not generate different agents if paramter given', function () {
```

Comment:
Please change the description to "should return the same agent if paramter is present"